### PR TITLE
docs: add issue template

### PR DESCRIPTION
**Pull request by bwanglzu**
Add issue templates.

### CODE REVIEW

**Review by cristianmtr — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/---missing---discussion-needed.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: "Create a documentation"
```
Comment (cristianmtr) [inline .github/ISSUE_TEMPLATE/---missing---discussion-needed.md]:
```suggestion
name: "Create documentation"
```

File: .github/ISSUE_TEMPLATE/---missing---discussion-needed.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+about: I’d like add/request a new documentation to Jina, before that I need some discussion.
```
Comment (cristianmtr) [inline .github/ISSUE_TEMPLATE/---missing---discussion-needed.md:3]:
```suggestion
about: I’d like to add/request new documentation to Jina, but I need some discussion.
```

File: .github/ISSUE_TEMPLATE/---incorrect---may-be-a-typo-.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+name: "Fix a typo or Rephrase a piece of text"
```
Comment (cristianmtr) [inline .github/ISSUE_TEMPLATE/---incorrect---may-be-a-typo-.md:2]:
```suggestion
name: "Fix a typo or rephrase a piece of text"
```

**Review by cristianmtr — CHANGES_REQUESTED** [review]
some stuff

File: .github/ISSUE_TEMPLATE/---missing---discussion-needed.md (hunk 2 above)
Comment (alexcg1) [reply .github/ISSUE_TEMPLATE/---missing---discussion-needed.md:3]: **for** Jina

File: .github/ISSUE_TEMPLATE/---missing---discussion-needed.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+labels: kind/docs
+assignees: ''
+
+---
+
+**Describe the documentation**
+<!-- A clear and concise description of what the feature is. -->
+
+**Your proposal**
+<!-- copy past your code/pull request link -->
```
Comment (alexcg1) [inline .github/ISSUE_TEMPLATE/---missing---discussion-needed.md:14]: paste

File: .github/ISSUE_TEMPLATE/---missing---discussion-needed.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+---
+<!-- Optional, but really help us locate the problem faster -->
```
Comment (alexcg1) [inline .github/ISSUE_TEMPLATE/---missing---discussion-needed.md:17]: can remove the bit about problem

File: .github/ISSUE_TEMPLATE/config.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
```
Comment (alexcg1) [inline .github/ISSUE_TEMPLATE/config.yml:2]: Add Slack link slack.jina.ai

File: .github/ISSUE_TEMPLATE/config.yml [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  - name: "📚 Read docs"
+    url: https://docs.jina.ai/
+    about: Find your solution from our documenations
```
Comment (alexcg1) [inline .github/ISSUE_TEMPLATE/config.yml:5]: documentation (no s)

**Review by alexcg1 — CHANGES_REQUESTED** [review]
Looking pretty good! Just minor tweakage

**Comment by FionnD** [comment]
cc @aga11313

**Review by FionnD — APPROVED** [review]

**Comment by bwanglzu** [comment]
I'll close this one, discuss if needed later.

**Pull request closed without merging**